app/helpers/navigation_helper.rb

```diff
@@ -144,14 +155,24 @@ def ontology_nav_metadata
 
   def repository_settings_nav(repository, current_page)
     pages = []
-    chain = resource_chain.last.is_a?(Ontology) ? resource_chain[0..-2] : resource_chain
+    chain =
+      if resource_chain.last.is_a?(Ontology)
+        resource_chain[0..-2]
+      else
+        resource_chain
+      end
     current_page = t("repository.#{current_page}")
-    pages << [t("repository.urlmaps"),  repository_url_maps_path(repository)]
-    pages << [t("repository.errors"),           repository_errors_path(repository)]
-    pages << [t("repository.permissions"),      [*chain, :permissions]] if can? :permissions, repository
-    pages << [t("repository.edit"), edit_repository_path(repository)]  if can? :edit, repository
+    pages << [t("repository.urlmaps"), repository_url_maps_path(repository)]
+    pages << [t("repository.errors"), repository_errors_path(repository)]
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.